### PR TITLE
Update TLS certificates document

### LINKED ISSUE #8404: Documentation for updating expired certificate incomplete
**Description**

The documentation for updating expired TLS Certificates is missing a required step. Following the steps as documented left me with a broken cluster. 

**Expected result**

The documentation for renewing an expired certificate should cover all the steps needed to successfully renew the cert.

**Actual result**
Kyma was still serving the old expired certificate after following the renewal instructions.

**Versions**
Kyma Version: 1.10.0
K8S Version: v1.14.8
Hyperscaler: Azure
Cluster Name: konduit-dev-5
Region: westus

**Steps to reproduce**

Following these docs:
https://kyma-project.io/docs/components/security/#tutorials-update-tls-certificate-steps

I did these steps:
 * Create a new certificate via cert-bot 
 * kubectl edit configmap -n kyma-installer owndomain-overrides
   * Add tlsCert and tlsKey content from new certificate
 * kubectl delete secret -n kyma-system apiserver-proxy-tls-cert
 * kubectl label installation/kyma-installation action=install
 * wait for update to finish
 * kubectl delete pod -n kyma-system -l app=console-backend-service

At this point the certificate in owndomain-overrides was up to date, but Kyma was still serving the old, expired certificate. And the secrets in the istio-system namespace for certificates were still referencing the old expired cert.

I understand after researching the failure that you need to delete the `net-global-overrides` configmap. This seems to be covered outside the regular docs:
 * https://github.com/kyma-project/kyma/pull/5696/files
 * https://kyma-community.slack.com/archives/CD1C94ZKP/p1586894867014800
   * `kubectl delete configmap -n kyma-install net-global-overrides`

### PR DESCRIPTION
**Pull request by superojla**
**Description**

Changes proposed in this pull request:

- Updated the procedure for the custom domain certificate
- Updated general procedure for Kyma update and console backend service restart

**Related issue(s)**
Resolves #8404

### CODE REVIEW

File: docs/security/08-02-update-tls-certificates.md [hunk 1]
```diff
@@ -22,16 +22,10 @@ The TLS certificate is a vital security element. Follow this tutorial to update
 
   >**CAUTION:** When you regenerate the TLS certificate for Kyma, the `kubeconfig` file generated through the Console UI becomes invalid. To complete these steps, use the admin `kubeconfig` file generated for the Kubernetes cluster that hosts the Kyma instance you're working on.
 
-  1. Edit the `owndomain-overrides` ConfigMap and replace the existing certificate and key found in the **global.tlsCrt** and **global.tlsKey** with a new, base64-encoded pair.
+  1. Delete the `net-global-overrides` ConfigMap and replace the existing certificate and key found in the **global.tlsCrt** and **global.tlsKey** with a new, base64-encoded pair.
```
Comment (colunira) [inline docs/security/08-02-update-tls-certificates.md:25]: I'm not sure where should I replace this existing certificate and key. In the update process? Create a new CM? It's a bit confusing.

Comment (kubadz) [reply docs/security/08-02-update-tls-certificates.md:25]: You should export the new values to env variables `TLS_CERT` and `TLS_KEY` which are used in the `kyma upgrade ...` command below. 

Anyway, I would recommend to move it to the next step, as it has nothing in common with removing the old ConfigMap.

File: docs/security/08-02-update-tls-certificates.md [hunk 2]
```diff
@@ -22,19 +22,46 @@ The TLS certificate is a vital security element. Follow this tutorial to update
...
+    ```
+
+  3. Trigger the update process. Run:
+
+    ```bash
+    kyma upgrade -s $KYMA_VERSION --domain $DOMAIN --tls-cert $TLS_CERT --tls-key $TLS_KEY
     ```
 
-    Delete the Secret that stores the expired certificate:
+    To watch the progress of the update, run:
```
Comment (kubadz) [inline docs/security/08-02-update-tls-certificates.md:46]: You don't need to use an additional command to watch the progress of the update. `kyma` CLI shows you the progress and terminates when Kyma is installed.

**Review by kubadz — CHANGES_REQUESTED** [review]

File: docs/security/08-02-update-tls-certificates.md (hunk 2 above)
Comment (superojla) [reply docs/security/08-02-update-tls-certificates.md]: Removed :)

**Review by kubadz — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]
LGTM 

fox smile

**Comment by sschommer** [comment]
Hi - is there a tutorial to rotate the certificate? We have a wildcard certificate that will expire in a few weeks. @colunira @parostatkiem-zz @superojla

**Comment by werdes72** [comment]
Hi @sschommer, depending on your environment, the certificate should be rotated automatically: https://github.com/gardener/cert-management#renewal-of-certificates
We don't have such tutorial but some of the links [here](https://kyma-project.io/docs/kyma/latest/04-operation-guides/security/sec-07-useful-links/) might be helpful.

**Comment by sschommer** [comment]
Hi - We are on Kyma 1.24.10 - so I think it is manual, plus we are using a custom domain wildcard cert. Do you know where instructions to rotate that kind of certificate are? Something like *.somecompany.com is the cert we have.
So we did something like this: https://kyma-project.io/docs/kyma/main/03-tutorials/00-security/sec-01-tls-certificates-security
I think we need the TLS or mTLS cert rotated - not sure which one we need to rotate.

**Comment by werdes72** [comment]
@sschommer, you have to rotate the certificate by yourself, when you have it, you can follow the same steps you linked to update it.

**Pull request merged**